### ISSUE TITLE
Text inputs within calendar (via 3rd party plugins) are not selectable

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2053]
Originally reported on Google Code with ID 1786

```
I've integrated Twitter Bootstrap Popover inside the fullCalendar.

When I click on event a popover is opened and show a text field with the event name
(that can be changed).

See this : <a href="http://tinypic.com?ref=1ob0aq" target="_blank"><img src="http://i48.tinypic.com/1ob0aq.jpg"
border="0" alt="Image and video hosting by TinyPic"></a>

I cannot select the text in the field and I notice that the problem is the "unselectable"
attribute on .fc-view

How can I make my field text "selectable"??

Thank you

```

Reported by `bellini.davide` on 2013-04-15 10:41:47

Imported with **1** stars.

**Comment by arshaw** [comment]
related to #2556

**Comment by arshaw** [comment]
still a problem: https://codepen.io/arshaw/pen/OJMmOoY?editable=true&editors=001
